src/main/resources/config/application-artemis.yml

```diff
@@ -69,3 +69,41 @@ artemis:
         clustering-url: http://localhost:8000/cluster
         segmentation-url: http://localhost:8080/segment
         secret: null
+info:
```

Comment:
This should only be in the `dev.yml` and `prod.yml`

Reply:
If I do not add it to artemis.yml, the integration test is failing because it cannot retrieve the filePatterns (as artemis is specified in the ActiveProfiles of AbstractSpringIntegrationTest). Do you know if there is a workaround that I can use for that?

Reply:
I think it would probably be best if you still remove this here, but add it to the `application.yml` in the test directory.